### PR TITLE
Review license

### PR DESCRIPTION
**Pull request by masensio**

### CODE REVIEW

File: src/com/owncloud/android/utils/TouchImageViewCustom.java [hunk 1]
```diff
@@ -1,13 +1,27 @@
...
- * By: Michael Ortiz
- * Updated By: Patrick Lackemacher
- * Updated By: Babay88
- * Updated By: @ipsilondev
- * Updated By: hank-cp
- * Updated By: singpolyma
- * -------------------
- * Extends Android ImageView to include pinch zooming, panning, fling and double tap zoom.
+/**
+ *   ownCloud Android client application
```
Comment (davivel) [inline src/com/owncloud/android/utils/TouchImageViewCustom.java]: Better move this file to third_parties package and keep it's license and copyright untouched; we made no change on it.

File: owncloud-android-library [hunk 2]
```diff
@@ -1 +1 @@
-Subproject commit 942971a202f7404b135b26795c1b545c390a023f
+Subproject commit 8c4834256b7d4f44822570973c03164d84586264
```
Comment (davivel) [inline owncloud-android-library]: Please, upgrade to commit in android-library/develop after merging license update.

**Comment by davivel** [comment]
Reviewed; please, check the comments.

**Comment by masensio** [comment]
Done :)

**Comment by davivel** [comment]
:+1:

**Pull request merged**